eng/scripts/Collect-Changelogs.ps1

```diff
@@ -0,0 +1,41 @@
+param($month)
```

Comment:
```suggestion
param([int]$month)
```

May as well have PowerShell validate it's a valid integer. With advanced functions (the `[CmdletBinding()]` stuff I've mentioned before, you can even use parameter validation to make sure it's 1 <= x <= 12. This really helps usability, rather than find out later that a script failed or - worse - corrupted data.

```powershell
[CmdletBinding()]
param(
  [Parameter(Mandatory=$true)]
  [ValidateRange(1, 12)]
  [int] $Month
)
```
)
```

Reply:
Done